plugins/scaffolder-backend/src/scaffolder/actions/types.ts

```diff
@@ -73,3 +73,11 @@ export type TemplateAction<Input extends JsonObject> = {
   };
   handler: (ctx: ActionContext<Input>) => Promise<void>;
 };
+
+export type BypassPullRequestAllowances =
+  | {
+      users?: string[];
+      teams?: string[];
+      apps?: string[];
+    }
+  | undefined;
```

Comment:
This should probably go right next to the github action, not in this general types file. Also, should not need to be `| undefined` _here_,  that would be up to the usage site to decide instead.

Reply:
I use this type in few files, so not sure it's a good idea to add same type to each file where it's used... Unless you have an idea where it should go specifically?

Reply:
just something like `actions/builtin/github/types.ts` or so, doesn't matter much but then it'll live with decidedly github related things

Reply:
> This should probably go right next to the github action, not in this general types file. Also, should not need to be `| undefined` _here_, that would be up to the usage site to decide instead.

I have tried to remove `| undefined`, but I get this error:
```
backstage/plugins/scaffolder-backend/src/scaffolder/actions/builtin/github/githubRepoPush.ts",

Argument of type 'BypassPullRequestAllowances | undefined' is not assignable to parameter of type 'BypassPullRequestAllowances'.
  Type 'undefined' is not assignable to type 'BypassPullRequestAllowances'.
```